include/ConfigManager.h

```diff
@@ -266,8 +272,8 @@ class LMMS_EXPORT ConfigManager : public QObject
 	ConfigManager(const ConfigManager & _c);
 	~ConfigManager();
 
-	void upgrade_1_1_90();
-	void upgrade_1_1_91();
+	void upgrade_0();
+	void upgrade_1();
```

Comment:
I don't think we need to rename these.

Reply:
From now on I believe this new naming will be used (`upgrade_` + config version, or something similar) because it makes it more explicit the methods being called per version. I thought changing the older ones to match this naming as well would make the code more organized, but I can revert the change if you think I should keep the old names.